packages/datadog_common_test/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: datadog_common_test
 description: Common helpers for testing Datadog libraries. Not meant to be a published package.
-version: 0.0.1
+version: 0.0.2
```

Comment:
No need to bump datadog_common_test. It's never published and is only a dev dependency included by path in the other packages.